src/mono/System.Private.CoreLib/src/System/Threading/ThreadPoolBoundHandle.Browser.Mono.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace System.Threading
 {
-    public sealed class ThreadPoolBoundHandle : IDisposable
+    public sealed partial class ThreadPoolBoundHandle : IDisposable
```

Comment:
I don't think this should be necessary, there might have been an issue before with some files getting incorrectly included when `FeaturePortableThreadPool == false`